pkg/asset/installconfig/gcp/validation_test.go

```diff
@@ -28,6 +29,10 @@ type editFunctions []func(ic *types.InstallConfig)
 var (
 	validNetworkName   = "valid-vpc"
 	validProjectName   = "valid-project"
+	validImage         = "valid-image"
+	projectID          = "project-id"
+	invalidImage       = "invalid-image"
+	mismatchedArch     = "mismatched-arch"
```

Comment:
Let's not pollute the global namespace. These are only used by `TestValidateMarketplaceImages` so they can be defined there.